front/cypress/e2e/project_page_timeline.cy.ts

```diff
@@ -148,29 +148,42 @@ describe('New timeline project', () => {
       .contains(phaseCurrentTitle);
     cy.get('.e2e-project-process-page').contains(phaseLongDescription);
 
+    // Verify that the description is collapsed by default
     cy.get('#e2e-project-phase-description-see-less-button').should(
       'not.exist'
     );
+
+    // Read more/less button functionality
     cy.get('#e2e-project-phase-description-read-more-button')
-      .should('exist')
+      .should('be.visible')
```

Comment:
We should assert be.visible in most cases rather than exist (should also help with general flakiness). Will post in the FE channel.